### PR TITLE
scene-builder: Add version 15.0.1

### PR DESCRIPTION
**Pull request by pratikpc**

### CODE REVIEW

File: bucket/scene-builder.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "version": "15.0.1",
+    "description": "Scene Builder is a visual, drag n drop, layout tool for designing JavaFX application user interfaces.",
+    "homepage": "https://gluonhq.com/products/scene-builder/",
+    "license": "BSD-3-Clause",
+    "url": "https://download2.gluonhq.com/scenebuilder/15.0.1/install/windows/SceneBuilder-15.0.1.msi",
+    "hash": "13b55c0832178b7756f0baaabc4b3bc8b2bc6687c0ad3ca55137256c57ff6a49",
+    "extract_dir": "SceneBuilder",
+    "bin": "SceneBuilder.exe",
```
Comment (Ash258) [inline bucket/scene-builder.json:9]: What are available command line arguments??

File: bucket/scene-builder.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "version": "15.0.1",
+    "description": "Scene Builder is a visual, drag n drop, layout tool for designing JavaFX application user interfaces.",
```
Comment (Ash258) [inline bucket/scene-builder.json:3]: Mentionning application name is nonsense when the manifest name is clear.

**Review by Ash258 — CHANGES_REQUESTED** [review]
- Does java runtime needs to be installed?
- Where is mentioned that it is compatible with java 11+??

image


image

**Comment by pratikpc** [comment]
> Does java runtime needs to be installed?

No. They add a local copy themselves in runtime folder which they use.

The Java 11 comment is for those who would want to run the JAR directly and not via the exe

File: bucket/scene-builder.json (hunk 1 above)
Comment (pratikpc) [reply bucket/scene-builder.json:9]: SceneBuilder.exe file-name

File: bucket/scene-builder.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+    "version": "15.0.1",
+    "description": "A visual, drag n drop, layout tool for designing JavaFX application user interfaces.",
+    "homepage": "https://gluonhq.com/products/scene-builder/",
+    "license": "BSD-3-Clause",
+    "url": "https://download2.gluonhq.com/scenebuilder/15.0.1/install/windows/SceneBuilder-15.0.1.msi",
```
Comment (Ash258) [inline bucket/scene-builder.json:6]: A

File: bucket/scene-builder.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        [
+            "SceneBuilder.exe",
+            "Scene Builder"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/gluonhq/scenebuilder"
+    },
+    "autoupdate": {
+        "url": "https://download2.gluonhq.com/scenebuilder/$version/install/windows/SceneBuilder-$version.msi"
```
Comment (Ash258) [inline bucket/scene-builder.json:20]: A

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/scene-builder.json (hunk 3 above)
Comment (pratikpc) [reply bucket/scene-builder.json:6]: I am a bit confused.

What's the issue?

Comment (Ash258) [reply bucket/scene-builder.json:6]: 64bit only...

File: bucket/scene-builder.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    "license": "BSD-3-Clause",
+    "architecture": {
+        "64bit": {
+            "url": "https://download2.gluonhq.com/scenebuilder/15.0.1/install/windows/SceneBuilder-15.0.1.msi",
+            "hash": "13b55c0832178b7756f0baaabc4b3bc8b2bc6687c0ad3ca55137256c57ff6a49",
+            "extract_dir": "SceneBuilder",
+            "bin": "SceneBuilder.exe",
+            "shortcuts": [["SceneBuilder.exe", "Scene Builder"]]
+        }
+    },
```
Comment (Ash258) [inline bucket/scene-builder.json:14]:
```suggestion
            "hash": "13b55c0832178b7756f0baaabc4b3bc8b2bc6687c0ad3ca55137256c57ff6a49"
        }
    },
    "extract_dir": "SceneBuilder",
    "bin": "SceneBuilder.exe",
    "shortcuts": [
        [
            "SceneBuilder.exe",
            "Scene Builder"
        ]
    ],
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Review by Ash258 — APPROVED** [review]

**Pull request merged**